fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/FedoraBinaryImpl.java

```diff
@@ -270,14 +276,20 @@ public String getFilename() {
     }
 
     @Override
-    public RdfStream getFixity(final IdentifierConverter<Resource, FedoraResource> idTranslator) {
-        return getFixity(idTranslator, getContentDigest(), getContentSize());
+    public RdfStream getFixity(final IdentifierConverter<Resource, FedoraResource> idTranslator,
+                               final String baseURL,
+                               final String userAgent,
+                               final EventBus eventBus) {
+        return getFixity(idTranslator, getContentDigest(), getContentSize(), baseURL, userAgent, eventBus);
     }
 
     @Override
     public RdfStream getFixity(final IdentifierConverter<Resource, FedoraResource> idTranslator,
                                final URI digestUri,
-                               final long size) {
+                               final long size,
+                               final String baseURL,
+                               final String userAgent,
+                               final EventBus eventBus) {
```

Comment:
Because instances of the core model types are not managed by the injection container. We could do that, but it would be a massive change in the core. @awoods and I have talked about it, but made no move.